indico/modules/events/editing/templates/emails/comment_notification.txt

```diff
@@ -1,11 +1,19 @@
 {% extends 'events/editing/emails/base_email.txt' %}
 
-{% block subject -%}{% trans %}New comment{% endtrans %}{%- endblock %}
+{% block subject -%}
+{% if programme_code -%}
+    {% trans %} New comment on contribution {{contribution_id}} with programme code {{programme_code}}{% endtrans %}
```

Comment:
I wouldn't directly shows users the contribution ID, since it's quite meaningless. If anything, maybe the `friendly_id`, but even that is more useful to managers than to submitters.
```suggestion
    {% trans %} New comment on {{programme_code}}{% endtrans %}
```
This should be enough